composer.json

```
@@ -53,11 +53,20 @@
             "psalm",
             "rector process --dry-run"
         ],
+        "sa": "@analyze",
         "ci": [
             "Composer\\Config::disableProcessTimeout",
+            "@cs-fix",
```

I don't know how (if?) other people use `composer ci` but I expect it to short-circuit if something needs changing:
```suggestion
            "@cs",
```